### PR TITLE
Add warning message near chat window about model hallucinations

### PR DESCRIPTION
**Pull request by Quarks-1**
Referencing issue #2756 , this is my current progress
image
Would love some feedback!

### CODE REVIEW

**Comment by Quarks-1** [comment]
This is the dark mode version:
image

**Comment by Quarks-1** [comment]
Also, I am currently using [this](https://assets.stickpng.com/images/5a81af7d9123fa7bcc9b0793.png) as the warning image, however, if there is an internal one that works better I'm happy to replace it.

**Review by notmd — COMMENTED** [review]
Thanks but I want to hold this until #2741 get merged

File: website/src/data/team.json
Comment (notmd) [inline website/src/data/team.json:198]: we probably dont want to add this team section, please move this to inside a component.

Comment (Quarks-1) [reply website/src/data/team.json:198]: Will do, thanks for the feedback!

**Comment by yk** [comment]
I'm not against this per se, but do we really have this low hopes for humanity?...

**Comment by andreaskoepf** [comment]
> I'm not against this per se, but do we really have this low hopes for humanity?...

I think this is really important as some outputs may look good, people are confused and see for example https://github.com/LAION-AI/Open-Assistant/issues/2751 . We should also state that the assistant has no internet access .. I saw at least 3 different videos which either in the video or later "as correction in the comments" stated that OA would have internet access .. probably mainly because our mission-statement and the docs page say this. Also more ordinary people without ML/NLP  experience or children might start using the system who have not heard about the hallucination problem.

I am not 100% happy with the current text proposal for the message, maybe could @olliestanley or someone else who is native speaker please make a proposal for the message?

**Comment by andreaskoepf** [comment]
btw I think we should also add a message to encourage users to give feedback with thumbs up/down for generated messages.

**Comment by Quarks-1** [comment]
@andreaskoepf Could we make the message to encourage users to give feedback a new issue since it is not directly related to this issue?

**Comment by olliestanley** [comment]
> Could we make the message to encourage users to give feedback a new issue since it is not directly related to this issue?

Yes that would be fine

**Comment by Quarks-1** [comment]
I've synced my fork and it seems like changes were pushed that don't pass pre-commit. These errors aren't related to the changes I've made @andreaskoepf

**Comment by olliestanley** [comment]
> I've synced my fork and it seems like changes were pushed to main that don't pass pre-commit. These errors aren't related to the changes I've made @andreaskoepf I've moved all of my changes to separate components so it should be good to go now!

pre-commit is passing on latest `main`. It looks like pre-commit's main issues for this PR are these:

```
./src/components/Chat/ChatSection.tsx
37:8  Error: 'Card' is not defined.  react/jsx-no-undef
38:10  Error: 'CardBody' is not defined.  react/jsx-no-undef
41:12  Error: 'Divider' is not defined.  react/jsx-no-undef
```

Other messages are just `Warning`

**Comment by notmd** [comment]
Hi, can you change the design to something like OAI does?
image

**Comment by andreaskoepf** [comment]
> Hi, can you change the design to something like OAI does? 

I think in a first step we should get this warning in. @notmd you can make a design proposal for something different, but IMO we have really important things for inference feedback collection which far more important .. we should discuss that via discord.

@Quarks-1 could you please address the pre-commit errors:
```
./src/components/Chat/ChatSection.tsx
1:1  Warning: Run autofix to sort these imports!  simple-import-sort/imports
37:8  Error: 'Card' is not defined.  react/jsx-no-undef
38:10  Error: 'CardBody' is not defined.  react/jsx-no-undef
41:12  Error: 'Divider' is not defined.  react/jsx-no-undef
```
thx!

**Comment by Quarks-1** [comment]
image

Here is the new warning to work with the codebase changes

**Comment by olliestanley** [comment]
> I am not 100% happy with the current text proposal for the message, maybe could @olliestanley or someone else who is native speaker please make a proposal for the message?

I can do this tomorrow, in the meantime we could merge this PR and I could submit a separate PR, either way is fine

**Comment by Quarks-1** [comment]
@olliestanley If there is a specific message you would like to have instead I am happy to replace it!

**Comment by andreaskoepf** [comment]
@olliestanley I think we could merge it now and you could maybe tweak the message a bit in a separate PR. @notmd if you are ok with the overall approach, e.g. using the `TeamMember` component here, it could be merged.

**Comment by olliestanley** [comment]
> @olliestanley If there is a specific message you would like to have instead I am happy to replace it!

As Andreas said, many people may not understand what "hallucination" is in this context. It's also not (yet) really true to say that safeguards have been put in place.

As a first draft I would consider something like:

`This Assistant is a demonstration version that does not have internet access. It may generate incorrect or misleading information. It is not suitable for important use cases or for giving advice.`

We can tweak this further later if needed.

**Comment by Quarks-1** [comment]
Sounds good, makes sense to make a separate PR then.

**Review by AbdBarho — APPROVED** [review]
We definitely need a design update, since this will block the majority of the screen for mobile users.

**Pull request merged**
